### ISSUE TITLE
Have you considered adding a New Issue option?

### ISSUE THREAD

**Issue by caboosesw** [Microsoft/vscode-github-issues-prs#31]
Would be helpful to not leave the editor

**Comment by chrmarti** [comment]
There is a '+' toolbar item next to the view title when the mouse cursor is in the view. That opens a new issue in the browser, so maybe that's not what you are looking for?

**Comment by chrmarti** [comment]
Only what's in the README in this repository. The extension uses the Git Credential Manager to figure out for which user to query for issues. You can also set that manually with the `"github.username"` setting for cases like yours.

**Comment by caboosesw** [comment]
Ok I tried finding where you want me to put this setting ... I tried with a git command but then it looks like there may be an elusive settings.json file that should be stored somewhere ... but no docs ... and I can't find in VSCode if there is a shortcut for configuring the extension

Halps?

**Comment by caboosesw** [comment]
Aha ... I believe you may be referring to the built in VSCode settings ... of which overriding user settings is the way I tried

{
    "github.username": "caboosesw",
    "github.repositories": "moira"
}

However ... because it is a private repository ... I suspect this won't get through? I may need to install the Git Credentials Mgr anyhow

**Comment by chrmarti** [comment]
Yes, that's currently a limitation.

**Comment by caboosesw** [comment]
If I get the Git Credentials Mgr going will I be able to access a private repository?

**Comment by chrmarti** [comment]
Yes, that would allow you to see all the repositories you have access to.
